src/app/features/theme-drawer/theme-list-item.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from 'react';
+import { Box, BoxProps, color, Flex, Stack } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+import { useDrawers } from '@app/common/hooks/use-drawers';
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { UserSelectedTheme, useThemeSwitcher } from '@app/common/theme-provider';
+import { CheckmarkIcon } from '@app/components/icons/checkmark-icon';
+
+interface ThemeListItemProps extends BoxProps {
+  theme: UserSelectedTheme;
+}
+export const ThemeListItem = ({ theme, ...props }: ThemeListItemProps) => {
```

Comment:
We could make a `ThemeListItemLayout` component that contains all the styles

Reply:
Good call, will do.